cilium/cmd/bpf_lb_list.go

```diff
@@ -15,8 +15,12 @@
 package cmd
 
 import (
+	"fmt"
```

Comment:
I think we should be able to dump both using some type of flag. Defaulting to `v2` is a good idea, but we should be able to introspect `v1` as well if some option, `--v1` is provided to this command as well.

Reply:
Noted. Sure, I will do it in a follow-up PR.